psi_cardinality/python/requirements.txt

```diff
@@ -0,0 +1,6 @@
+black>=10.3b0
+bumpversion>=0.5.3
+flake8
+pytest>=5.4.1
+pytest-benchmark
+setuptools>=41.0.0
```

Comment:
This aren't actually needed for the python package but for testing and it might help to have the file named requirements_dev.txt to be sure that it's not misleading